app/poros/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+
+  attr_reader :title, :author, :description, :thumbnail, :isbn, :category
+
+  def initialize(attributes)
+    @title = attributes[:items].first[:volumeInfo][:title]
```

Comment:
The stuff to the right of the equals sign will change when we customize how our book data is serialized.